presto-product-tests/src/main/java/io/prestosql/tests/cassandra/TestSelect.java

```diff
@@ -313,6 +313,25 @@ public void testSelectClusteringMaterializedView()
         onCassandra(format("DROP MATERIALIZED VIEW IF EXISTS %s.%s", KEY_SPACE, mvName));
     }
 
+    @Test(groups = CASSANDRA)
+    public void testSelectTupleType()
+    {
+        String tableName = "insert_tuple_table";
+
+        onCassandra(format("DROP TABLE IF EXISTS %s.%s", KEY_SPACE, tableName));
+
+        onCassandra(format("CREATE TABLE %s.%s (key int, value frozen<tuple(int, int)>, PRIMARY KEY (key))",
```

Comment:
Let's add more type as above comment. Also `tuple(int, int)` will throw parse-error. Please use angle brackets `<` and `>`